skrub/_select_cols.py

```diff
@@ -109,6 +114,11 @@ class DropCols(TransformerMixin, BaseEstimator):
         ``str``: ``"col_name"`` is the same as ``["col_name"]``. See the
         :ref:`selectors <user_guide_selectors>` user guide for more info on selectors.
 
+    See Also
+    --------
+    SelectCols : Selecting cols by name, dtypes, or general skrub selectors.
+    Cleaner: Can be used to drop columns with too many NaNs.
```

Comment:
```suggestion
    Cleaner: Can be used to drop columns with too many nulls (or NaNs).
```
not sure about the wording (maybe just nulls?), but the Cleaner drops either, not only NaNs